paddlenlp/transformers/model_utils.py

```diff
@@ -964,13 +1016,9 @@ def _load_pretrained_model(
         add_prefix_to_model = has_prefix_module and not expects_prefix_module
 
         if remove_prefix_from_model:
-            expected_keys_not_prefixed = [
-                s for s in expected_keys if not s.startswith(prefix)
-            ]
-            expected_keys = [
-                ".".join(s.split(".")[1:]) if s.startswith(prefix) else s
-                for s in expected_keys
-            ]
+            # expected_keys_not_prefixed = [s for s in expected_keys if not s.startswith(prefix)]
+            # TODO: is this line required?
```

Comment:
@wj-Mcat 
this line was marked as unused variable definition. Please check whether it's safe to remove

Reply:
this variable is not used, so it shoule be removed.

Reply:
I have removed it in another pr.

Reply:
ok. i'll wait until you merge your PR in.